include/utils.h

```diff
@@ -1014,6 +1014,71 @@ template <typename T> inline void normalize(T *arr, size_t dim)
         arr[i] = (T)(arr[i] / sum);
     }
 }
+template <typename T> inline std::vector<T> readFileLinesInVector(const std::string &filename, bool unique = false)
```

Comment:
could we call this read_file_to_vector_of_strings

Reply:
Also, do we really need to parse as strings as opposed to labelT (which could be set to uint32, uint64)

Reply:
We could do labelT but, I
 was thinking of a more generic function. Let me translate the file reading logic to respect the template type.